<launch>
    <node pkg="task3" type="task3_ring_detection" name="ring_detection" output="screen">
        <param name="depth" type="string" value="/camera/depth/image_raw"/>
        <param name="rgb" type="string" value="/camera/rgb/image_raw"/>
        <param name="cam_info" type="string" value="normal"/>
        <param name="debug" type="bool" value="false"/>
    </node>

    <node pkg="task3" type="task3_cylinder_detection" name="cylinder_detection" output="screen">
    </node>

    <node pkg="task3" type="face_posters.py" name="face_posters" output="screen">
    </node>

    <node pkg="task3" type="dialogue.py" name="dialogue" output="screen">
    </node>

    <node pkg="task3" type="task3_ground_ring_detection" name="ground_ring_detection" output="screen">
        <param name="depth" type="string" value="/arm_camera/depth/image_raw"/>
        <param name="rgb" type="string" value="/arm_camera/rgb/image_raw"/>
        <param name="cam_info" type="string" value="arm"/>
        <param name="debug" type="bool" value="false"/>
    </node>
</launch>
